jaffle_shop/models/warehouse/wh_customers.sql

```diff
@@ -1,6 +1,6 @@
 with customers as (
 
-    select * from {{ ref('stg_customers') }}
+    select * from {{ ref('stg_customers_pii') }}
 
 ),
 
```

Comment:
I've missed this in other PRs - SELECT * from cte's should generally not be used (only select columns that will be used ) + because this model uses PII, it should be renamed `wh_customers_pii` and put in the sensitive schema. No need to do this.

Reply:
Sure, I'll revert this change then to just select from stg_customers